integration-test/840-normalize-place-kind.py

```diff
@@ -68,31 +68,31 @@
 
 # ne capitals
 assert_has_feature(
-    16, 55887, 25378, 'places',
+    7, 109, 49, 'places',
     { 'id': int, 'name': 'Seoul',
       'kind': 'locality', 'capital': True})
```

Comment:
This will start to fail soon as `'capital': True` now needs to be `'country_capital': True` from Rob's PR in https://github.com/tilezen/vector-datasource/pull/1003/files#diff-a53ac4538d3cfcaeb972c98bcaaa5e84R40 and https://github.com/tilezen/vector-datasource/pull/1003/files#diff-d6e325987b23272d73f3da7d12c50972R65.